e2e_playwright/st_tabs_selection.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
+
+st.subheader("Control Panel", divider="blue")
+
+if "tabs" not in st.session_state:
+    st.session_state["tabs"] = ["Tab 1", "Tab 2"]
+if "add_tab" not in st.session_state:
+    st.session_state["add_tab"] = False
+if "remove_1" not in st.session_state:
+    st.session_state["remove_1"] = False
+if "remove_2" not in st.session_state:
+    st.session_state["remove_2"] = False
+if "change" not in st.session_state:
+    st.session_state["change"] = False
+
+
+def onClick1():
```

Comment:
Do these need to be camel case, or can they be changed to snake case to follow our usual formatting?